Project Management Process.md

```diff
@@ -0,0 +1,287 @@
+
+TODO: a clear example
+
+### Project Roadmap
+The milestones are tracked in project's roadmap. The project lead owns the roadmap and is responsible for generating, updating and tracking the milestones. To make sure the roadmap is up to date, the project lead should go through the current milestones on weekly basis.
+
+The Roadmap Document contains the following information:
+- List of current milestones
+  - Sequentally ordered, eg. by expected due date (where applicable)
+  - Next upcoming milestone at the top
+- Current status of the project, ie. milestone progress
+- List of previous milestones
+
+The roadmap can be a high-level overview of milestones and project's progress, so details such as list of goals per milestone can be omitted. However, if omitted, the roadmap must contain links to the detailed break down.
+
+Examples:
+- [js-ipfs roadmap](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#ipfs-javascript-implementation-roadmap)
+- [Orbit roadmap](https://github.com/haadcode/orbit/blob/master/ROADMAP.md#orbit---roadmap)
+
+### Organization Roadmap
+An overview of all projects across the organization is tracked in organization's roadmap. This is similar to a project roadmap, but collects all milestones from all projects into one place.
+
+Organization roadmap contains the following information:
+- List of all projects
+- List of all current milestones and their progress in each project
+- List of links to old milestones
+
+The organization roadmap is owned by project management and project managers are responsible for generating and keeping the roadmap up to date. The roadmap should be updated in the quarterly planning meeting as old milestones are reviewed and the project leads commit to new milestones.
+
+TODO: a clear example
+
+## Workflow
+
+### Pipelines
+The IPFS Project Management Process uses Pipelines to process goals towards a release. Pipelines are a simple way to describe the workflow of a project and move Goals between the different stages in their lifecycle. 
+
+#### Stages
+Pipelines are sequential stages through which a Goal goes during its lifetime. The different stages are broken down per the workflow of each project. On high level the stages can be described as: `Not Done ──> In Progress ──> Done`. Different projects might have different stages in their development pipeline and it is up to the project lead to identify and define the stages her project uses.
```

Comment:
i dislike this characterization of "pipeline" here because 
- it doesn't fit the general idea
- i'm going to use "pipelines" to track things _beyond_ individual Goals' "not done -> in progress -> done".
- maybe use "Progress Pipelines" to denote the pipelines that track progress according to this model (goals, etc?). OR just mention the "Kanban Pipeline" ?


For example, i will create concrete pipelines to handle publishing talk materials, which will look like this: https://ipfs.io/ipfs/Qmb4EXZ8uWBrFGpa8GDvFkvvtL5GN2TSuq2gtMXc6zwUng/#README.md -- there's multiple pipelines, that will span multiple people, with multiple pieces of work, across multiple repos. I've no idea yet how this kind of pipeline would be instantiated (mechanics), but this is a very different sort of pipeline that does not follow the model of "in progress

I can imagine other pipelines that would fork (i.e. items go different directions depending on what they are).